rtslib/utils.py

```diff
@@ -511,6 +511,32 @@ def set_parameters(obj, param_dict, err_func):
         except RTSLibError as e:
             err_func(str(e))
 
+def get_enable(so):
+    '''
+    @return: True if the StorageObject is configured, else returns False
```

Comment:
In rtslib naming, StorageObject or "so" means an instance of a backstore. Since this commit is all about generalizing the enable/disable code to both storageobjects and TPGs, it would be better to make reference in this code to CFSNode or "node" -- the class both SOs and TPGs are derived from.

In fact, thinking about this more, maybe both of these functions should just be methods on CFSNode?

Reply:
I am still getting used to object oriented programming :) One question about this comment with that in mind.

Are fields/properties/functions on CFSNode supposed to be something that are supported by all classes that are derived from that parent class? I am asking because other classes like Target, NodeACL, etc do not have/support the enable/disable concept. For those type of classes would you override the CFSNode functions and just do something like always return True/success?

Reply:
I don't think you'd need to, because it looks like the code already handles this case -- if a node doesn't have an `enable` file to read, it is considered enabled. The other part is `set_enable` -- I think it also already does what we'd want -- an attempt to disable a a node w/o `enable` should raise an error.